source-academy/cadet-frontend
# Add ability to disable the frontend during specified periods

### Description

Add the ability to disable the frontend during specified periods.

![image](https://user-images.githubusercontent.com/235432/91609093-569b8900-e9a9-11ea-9b5c-4837596a8299.png)

### Type of change


### How to test

Configure a disablement period locally and see that it renders the Disabled page during the disablement period, and renders the original page after. (Note that any state in the original page is lost, of course.)

### Checklist
